### PR TITLE
Text for "No Entries" appears in spanish for EN language

### PR DESCRIPTION
**Pull request by franciscoghersi**

### CODE REVIEW

File: src/DonateTo.WebApplication/ClientApp/src/app/home/components/donation-request/list/donation-requests-list.component.ts [hunk 1]
```diff
@@ -1,47 +1,70 @@
-import { Component, Input, OnInit, Output } from '@angular/core';
+import { DonationRequestModel } from './../../../../shared/models/donation-request.model';
```
Comment (Faculewis) [inline src/DonateTo.WebApplication/ClientApp/src/app/home/components/donation-request/list/donation-requests-list.component.ts:1]: import from the index file that is in the models' folder

File: src/DonateTo.WebApplication/ClientApp/i18n/en.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
   "GridEmptyLabel": "No available items",
   "MoreDetail": "More details",
   "SearchPlaceHolder": "input search text",
+  "NoEntries": "No entries",
```
Comment (Faculewis) [inline src/DonateTo.WebApplication/ClientApp/i18n/en.json:28]: Let this as a default "NoEntries" traslation but add the same item inside RequestDonation and the text should be "No new donation requests"

File: src/DonateTo.WebApplication/ClientApp/src/app/home/components/donation-request/list/donation-requests-list.component.html [hunk 3]
```diff
@@ -1,27 +1,33 @@
 <nz-list
-  [nzDataSource]="donationRequests"
+  [nzDataSource]="isLoading ? mockData : (homeSandbox.donationRequests$ | async)"
```
Comment (Faculewis) [inline src/DonateTo.WebApplication/ClientApp/src/app/home/components/donation-request/list/donation-requests-list.component.html:2]: Change `(homeSandbox.donationRequests$ | async)` with `donationRequests`

File: src/DonateTo.WebApplication/ClientApp/i18n/es.json [hunk 4]
```diff
@@ -8,7 +8,8 @@
     "ResourcesNeeded": "Recursos necesitados",
     "Priority": "Prioridad",
     "CreatedOn": "Creado",
-    "Finisheson": "Termina"
+    "Finisheson": "Termina",
```
Comment (guillemurano) [inline src/DonateTo.WebApplication/ClientApp/i18n/es.json:11]: Add article to fit finishes on translation to be "Termina el"

**Comment by aupace** [comment]
Fix verified.

**Pull request merged**